src/sass/app/sell_popup.scss

```diff
@@ -120,6 +120,12 @@
                         font-weight: normal;
                     }
                 }
+                @media screen and (max-width: 480px) {
```

Comment:
nit, please put the media screen for the respective classs , for example view-chart class is inside of `#sell_details_chart_wrapper` .